### PR TITLE
113 - Work Items Persistence

### PR DESCRIPTION
**Pull request by codyseibert**
More tests will be in the next PR

- adding the implementation to store / fetch work items from dynamodb

### CODE REVIEW

File: efcms-service/storage/fixtures/efcms.json [hunk 1]
```diff
@@ -44,6 +48,32 @@
...
+        "message": "here is a random message! yay!"
+     } 
+    ],
+    "sentBy": "respondent",
+    "createdAt": "2018-12-12T23:45:02.474Z",
+    "updatedAt": "2018-12-12T23:45:02.474Z",
+    "assigneeId": "docketclerk",
+    "docketNumber": "101-18",
+    "caseId": "1a6fb0fb-5f93-479a-9235-7eba43005800",
+    "assigneName": "John Smith (Docketclerk)",
```
Comment (kkoskelin) [inline efcms-service/storage/fixtures/efcms.json]: typo? "assigneName"?

File: shared/src/business/entities/WorkItem.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+      .date()
+      .iso()
+      .required(),
+    assigneeId: joi.string().required(),
+    docketNumber: joi.string().required(),
+    caseId: joi
+      .string()
+      .uuid(uuidVersions)
+      .required(),
+    assigneName: joi.string().required(),
```
Comment (kkoskelin) [inline shared/src/business/entities/WorkItem.js]: same typo?

File: shared/src/business/useCases/respondent/fileStipulatedDecision.interactor.js [hunk 3]
```diff
@@ -13,15 +14,23 @@ exports.fileStipulatedDecision = async ({
...
         assigneeId: 'docketclerk',
         docketNumber: caseToUpdate.docketNumber,
-        //document is added later
+        messages: [
+          {
+            message: `Stipulated Decision submitted`,
+            createdAt: new Date().toISOString(),
+          },
+        ],
+        assigneName: 'Docket Clerk',
```
Comment (kkoskelin) [inline shared/src/business/useCases/respondent/fileStipulatedDecision.interactor.js]: am I wrong about this being a typo?  :)

Comment (sadlerw) [reply shared/src/business/useCases/respondent/fileStipulatedDecision.interactor.js]: you are not wrong

**Review by sadlerw — APPROVED** [review]

**Pull request merged**